From: John Villalovos <jvillalo@redhat.com>
Date: Fri, 5 Dec 2008 11:27:17 -0500
Subject: [agp] update the names of some graphics drivers
Message-id: 20081205162717.GA5557@linuxjohn.usersys.redhat.com
O-Subject: [RHEL 5.3] BZ 472438 2nd post: Add final branding strings for Cantiga graphics (Montevina) to graphics driver
Bugzilla: 472438
RH-Acked-by: Jiri Pirko <jpirko@redhat.com>
RH-Acked-by: Peter Martuccelli <peterm@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Same as before, except I changed the "Intel?" to "Intel(r)".  No other changes
were made.

This is for:
https://bugzilla.redhat.com/show_bug.cgi?id=472438

I was asked by Peter Martucelli to provide this patch before 5pm Friday.

This was brew built and status can be found at:
https://brewweb.devel.redhat.com/taskinfo?taskID=1599616

Upstream commit can be seen:
Upstream checked out kernel:  $ git log -p --no-walk 99d32bd5c7b1caa05d1fe3c89b08aabd459bc12a
or
http://git.kernel.org/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commitdiff;h=99d32bd5c7b1caa05d1fe3c89b08aabd459bc12a

This has been changed: On line 44 of the patch, the "Intel?" is part of
upstream so I kept it.

This patch is fairly simple, the main purpose of this patch is that it changes
the string from "Intel Integrated Graphics Device" to "Mobile Intel? GM45
Express".  The #defines are also updated to more clearly represent that they
are for the GM45 device.

diff --git a/drivers/char/agp/intel-agp.c b/drivers/char/agp/intel-agp.c
index 8cbcea5..8158400 100644
--- a/drivers/char/agp/intel-agp.c
+++ b/drivers/char/agp/intel-agp.c
@@ -30,8 +30,8 @@
 #define PCI_DEVICE_ID_INTEL_Q35_IG          0x29B2
 #define PCI_DEVICE_ID_INTEL_Q33_HB          0x29D0
 #define PCI_DEVICE_ID_INTEL_Q33_IG          0x29D2
-#define PCI_DEVICE_ID_INTEL_IGD_HB          0x2A40
-#define PCI_DEVICE_ID_INTEL_IGD_IG          0x2A42
+#define PCI_DEVICE_ID_INTEL_GM45_HB         0x2A40
+#define PCI_DEVICE_ID_INTEL_GM45_IG         0x2A42
 #define PCI_DEVICE_ID_INTEL_IGD_E_HB        0x2E00
 #define PCI_DEVICE_ID_INTEL_IGD_E_IG        0x2E02
 #define PCI_DEVICE_ID_INTEL_Q45_HB          0x2E10
@@ -61,7 +61,7 @@
 #define IS_G4X (agp_bridge->dev->device == PCI_DEVICE_ID_INTEL_IGD_E_HB || \
 		agp_bridge->dev->device == PCI_DEVICE_ID_INTEL_Q45_HB || \
 		agp_bridge->dev->device == PCI_DEVICE_ID_INTEL_G45_HB || \
-		agp_bridge->dev->device == PCI_DEVICE_ID_INTEL_IGD_HB)
+		agp_bridge->dev->device == PCI_DEVICE_ID_INTEL_GM45_HB)
 
 /* Intel 815 register */
 #define INTEL_815_APCONT	0x51
@@ -991,7 +991,7 @@ static unsigned long intel_i965_mask_memory(struct agp_bridge_data *bridge,
 static void intel_i965_get_gtt_range(int *gtt_offset, int *gtt_size)
 {
 	switch (agp_bridge->dev->device) {
-	case PCI_DEVICE_ID_INTEL_IGD_HB:
+	case PCI_DEVICE_ID_INTEL_GM45_HB:
 	case PCI_DEVICE_ID_INTEL_IGD_E_HB:
 	case PCI_DEVICE_ID_INTEL_Q45_HB:
 	case PCI_DEVICE_ID_INTEL_G45_HB:
@@ -2074,10 +2074,10 @@ static int __devinit agp_intel_probe(struct pci_dev *pdev,
 		bridge->driver = &intel_7505_driver;
 		name = "E7205";
 		break;
-	case PCI_DEVICE_ID_INTEL_IGD_HB:
-		if (find_i830(PCI_DEVICE_ID_INTEL_IGD_IG)) {
+	case PCI_DEVICE_ID_INTEL_GM45_HB:
+		if (find_i830(PCI_DEVICE_ID_INTEL_GM45_IG)) {
 			bridge->driver = &intel_i965_driver;
-			name = "Intel Integrated Graphics Device";
+			name = "Mobile Intel(r) GM45 Express";
 		} else
 			bridge->driver = NULL;
 		break;
@@ -2267,7 +2267,7 @@ static struct pci_device_id agp_intel_pci_table[] = {
 	ID(PCI_DEVICE_ID_INTEL_G33_HB),
 	ID(PCI_DEVICE_ID_INTEL_Q35_HB),
 	ID(PCI_DEVICE_ID_INTEL_Q33_HB),
-	ID(PCI_DEVICE_ID_INTEL_IGD_HB),
+	ID(PCI_DEVICE_ID_INTEL_GM45_HB),
 	ID(PCI_DEVICE_ID_INTEL_IGD_E_HB),
 	ID(PCI_DEVICE_ID_INTEL_Q45_HB),
 	ID(PCI_DEVICE_ID_INTEL_G45_HB),
